index.html

```diff
@@ -39,23 +46,200 @@ <h2>I'm Jay</h2>
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons3">
+          <li><button class="li_buttons3">html</button></li>
+          <li><button class="li_buttons3">bootstrap</button></li>
+          <li><button class="li_buttons3">ruby</button></li>
+        </ul>
+        <button class="card3button">See Project</button>
+      </div>
+
+      <div class="card4">
+        <h2 class="card4_title">Professional Art</h2>
+        <h2 class="card4_title4">Printing Data</h2>
+        <p class="card4_paragraph">A daily section of privately personalized reads; no accounts or signups required
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons4">
+          <li><button class="li_buttons4">html</button></li>
+          <li><button class="li_buttons4">bootstrap</button></li>
+          <li><button class="li_buttons4">ruby</button></li>
+        </ul>
+        <button class="card4button">See Project</button>
+      </div>
+
+      <div class="card5">
+        <h2 class="card5_title">Professional Art</h2>
+        <h2 class="card5_title5">Printing Data</h2>
+        <p class="card5_paragraph">A daily section of privately personalized reads; no accounts or signups required
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons5">
+          <li><button class="li_buttons5">html</button></li>
+          <li><button class="li_buttons5">bootstrap</button></li>
+          <li><button class="li_buttons5">ruby</button></li>
+        </ul>
+        <button class="card5button">See Project</button>
+      </div>
+
+      <div class="card6">
+        <h2 class="card6_title">Professional Art</h2>
+        <h2 class="card6_title6">Printing Data</h2>
+        <p class="card6_paragraph">A daily section of privately personalized reads; no accounts or signups required
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons6">
+          <li><button class="li_buttons6">html</button></li>
+          <li><button class="li_buttons6">bootstrap</button></li>
+          <li><button class="li_buttons6">ruby</button></li>
+        </ul>
+        <button class="card6button">See Project</button>
+      </div>
+
+      <div class="card7">
+        <h2 class="card7_title">Professional Art</h2>
+        <h2 class="card7_title7">Printing Data</h2>
+        <p class="card7_paragraph">A daily section of privately personalized reads; no accounts or signups required
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons7">
+          <li><button class="li_buttons7">html</button></li>
+          <li><button class="li_buttons7">bootstrap</button></li>
+          <li><button class="li_buttons7">ruby</button></li>
+        </ul>
+        <button class="card7button">See Project</button>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ]  [REQUIRED] You forget to adjust the space between the project section and the about section. As the first reviewer requested Please try to follow the design as closely as possible. You did great at improving it from the previous. As a developer, you will have to reproduce a given design as strictly as possible. Sticking to the Figma template as much as possible (e.g. font, colors, images, text, margins) will help you to have a wonderful website.  Check the screenshots below to see the difference :arrow_down: 

#### Figma template
![Screenshot from 2022-08-19 09-51-17](https://user-images.githubusercontent.com/6615394/185593800-a6acc2c5-a0de-4884-b73e-7102db43d13b.png)

#### Your design
![Screenshot from 2022-08-19 09-50-50](https://user-images.githubusercontent.com/6615394/185593808-b0269770-45db-40ca-9d45-df7ae3f7a9b3.png)